process/setup_config.py

```diff
@@ -0,0 +1,181 @@
+################################################################################
+# Script: setup_config.py
+# Description: This script defines project parameters and prepare configuration file for all study regions
+# All the cities should run this script first to create configuration file in json
+# before running the sample point and aggregation scripts.
+
+# Two major outputs:
+# 1. study region configuration json (e.g. odense.json; phoenix.json)
+# 2. cities.json
+
+################################################################################
+import json
+import os
+import sys
+import time
+
+# define project parameters
+# list of cities that are needed to be set up
+cities = ['adelaide', 'auckland', 'baltimore', 'bangkok', 'barcelona', 'belfast', 'bern', 'chennai',
+          'cologne', 'ghent', 'graz', 'hanoi', 'hong_kong', 'lisbon', 'melbourne', 'mexico_city',
+          'odense', 'olomouc', 'phoenix', 'sao_paulo', 'seattle', 'sydney', 'valencia', 'vic']
+
+# list of countries regions, follow the order of list of cities above
+regions = ['au', 'nz', 'us', 'th', 'es', 'gb', 'ch', 'in',
+           'de', 'be', 'at', 'vn', 'hk', 'pt', 'au', 'mx',
+           'dk', 'cz', 'us', 'br', 'us', 'au', 'es', 'es']
+
+# list of UTM zone, follow the order of list of cities above
+to_crs = [7845, 2193, 32618, 32647, 25831, 29902, 32633, 32644,
```

Comment:
I might recommend combining `cities`, `regions`, and `to_crs` lists into a single list of dicts, like `cities = [{'name':'adelaide', 'region':'au', 'crs':'epsg:7845'}, etc.]`